enigma-cli/src/main/java/cuchaz/enigma/command/CheckMappingsCommand.java

```diff
@@ -62,7 +63,7 @@ public static void run(Path fileJarIn, Path fileMappings) throws Exception {
 					.count();
 			if (packages > 1) {
 				error = true;
-				System.err.println("ERROR: Must be in one package:\n" + partition.stream()
+				Logger.error("Must be in one package:\n" + partition.stream()
```

Comment:
You should prefer interpolation over string concat, to avoid computing this when certain types of logging might not be enabled: https://tinylog.org/v2/logging/#text-with-arguments -- you can wrap the `partition.stream()` bit into a lambda to defer computing until you know the logging will actually happen.